src/components/Pagination/Pagination.scss

```diff
@@ -1,47 +1,212 @@
 @import "./../../base/base.scss";
 
-.nitrozen-pagination-container {
+.n-pagination-container {
   font-family: $PrimaryFont;
   color: $TypographyPrimaryColor;
-  padding: 1.2rem;
+  border: 1px solid $ColorPrimaryGrey60;
+  border-radius: 12px;
+  padding: 12px 12px;
 
-  .nitrozen-pagination {
+  .n-pagination {
     display: flex;
     font-size: 1.4rem;
     color: $LabelColor;
     font-size: 1.4rem;
     font-weight: 400;
-
-    .nitrozen-pagination__left {
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 12px;
+    .mobile_view {
+      display: none !important;
+      @media only screen and (max-width: $sm-min) {
+        display: flex !important;
```

Comment:
Are we sure using `!important` will not impact any other styles? Please remove if possible

Reply:
It is required and it ll not affecting to any other styles.

Reply:
It is required and it ll not affecting to any other styles.